src/utils.ts

```diff
@@ -29,6 +29,13 @@ export interface CleanedAlbumTrack {
   songUrl: string,
 }
 
+export interface Playlist {
+  id: number,
+  name: string,
+  isSaved: boolean,
+  tracks: CleanedAlbumTrack[]
+}
+
```

Comment:
This interface feels much more appropriate and focused on what we are creating. As mentioned earlier, it may be worthwhile to also changed the name of the interface 'CleanedAlbumTrack' to also be more focused on what it is supposed to be.